client/pages/item/_itemId/index.vue

```diff
@@ -218,17 +194,13 @@
               </v-row>
               <v-row align="center">
                 <v-col
-                  :cols="twoColsInfoColumn.lCols"
-                  :sm="twoColsInfoColumn.lSm"
-                  :class="twoColsInfoColumn.lClass"
+                  :cols="12"
+                  :sm="2"
+                  class="mt-sm-3 py-sm-0 px-0 text-truncate"
```

Comment:
I actually think the previous way was more readable. I've always found the flexbox sizing extremely annoying to read.

Reply:
Eh, it's a habit to take, imo. This way is similar to what you'd have with Tailwind or other CSS utilities.

I generally find having to refer to code when I want to change more annoying. Plus if I want to change one, I'd have to refactor everything.

Reply:
I think the biggest problem I have here (and with lots of other implementations using class-based css frameworks) is that these won't be standardized across all the page layouts. There isn't a good solution for pages like there is for components besides making them as generic as possible, but I figured it was worth mentioning.